x-pack/plugins/cases/public/components/user_actions/index.tsx

```diff
@@ -248,7 +217,9 @@ export const UserActions = React.memo(
 
     const { permissions } = useCasesContext();
 
-    const bottomActions = permissions.create
+    const showCommentEditor = permissions.create && filterOptions !== 'action'; // add-comment markdown is not visible in History filter
+
+    const bottomActions = showCommentEditor
```

Comment:
Could you please add a unit test to test if the comment editor is not being shown when the filter option is `action`?